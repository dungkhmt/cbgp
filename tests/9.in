50 50
50 100
32 49
43 39
46 24
24 48
43 34
45 8
34 4
33 44
6 0
28 4
36 7
36 13
17 5
46 11
36 9
42 6
15 8
20 1
4 5
4 6
8 5
18 17
35 11
12 17
47 11
38 20
45 14
49 4
41 46
30 21
6 2
31 1
22 16
20 27
8 9
23 17
31 12
29 24
13 26
48 39
29 3
25 19
22 2
30 19
12 44
45 10
40 16
31 21
37 12
1 2
45 3
14 3
37 0
49 2
42 22
35 14
15 9
33 17
5 34
42 16
13 1
8 1
14 11
42 27
19 0
45 7
6 1
29 11
30 25
23 37
39 32
32 4
21 19
18 23
17 0
31 19
31 0
46 29
2 3
23 12
26 7
43 32
48 11
13 9
42 20
26 3
35 47
9 7
41 48
45 9
33 31
39 2
33 8
33 12
13 7
49 39
38 42
23 0
46 48
32 34
1
48 8 10 5 34 22 38 26 62 16 57 0 42 14 36 28 2 0 33 31 0 0 48 30 15 2 32 27 43 31 31 28 37 18 55 1 51 3 43 12 28 15 45 9 34 19 51 4 48 26 42 10 37 27 33 15 76 1 44 26 44 10 12 4 55 26 9 1 82 0 44 32 34 29 50 5 31 18 53 25 35 20 49 27 32 17 55 27 12 3 37 37 48 29 45 33 52 26 54 24 
