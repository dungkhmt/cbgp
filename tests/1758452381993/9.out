(32, 49), (43, 39), (46, 24), (24, 48), (43, 34), (45, 8), (34, 4), (33, 44), (6, 0), (28, 4), (36, 7), (36, 13), (17, 5), (46, 11), (36, 9), (42, 6), (15, 8), (20, 1), (4, 5), (4, 6), (8, 5), (18, 17), (35, 11), (12, 17), (47, 11), (38, 20), (45, 14), (49, 4), (41, 46), (30, 21), (6, 2), (31, 1), (22, 16), (20, 27), (8, 9), (23, 17), (31, 12), (29, 24), (13, 26), (48, 39), (29, 3), (25, 19), (22, 2), (30, 19), (12, 44), (45, 10), (40, 16), (31, 21), (37, 12), (1, 2), (45, 3), (14, 3), (37, 0), (49, 2), (42, 22), (35, 14), (15, 9), (33, 17), (5, 34), (42, 16), (13, 1), (8, 1), (14, 11), (42, 27), (19, 0), (45, 7), (6, 1), (29, 11), (30, 25), (23, 37), (39, 32), (32, 4), (21, 19), (18, 23), (17, 0), (31, 19), (31, 0), (46, 29), (2, 3), (23, 12), (26, 7), (43, 32), (48, 11), (13, 9), (42, 20), (26, 3), (35, 47), (9, 7), (41, 48), (45, 9), (33, 31), (39, 2), (33, 8), (33, 12), (13, 7), (49, 39), (38, 42), (23, 0), (46, 48), (32, 34), 
numberIntersection=0 
minDistance=2.82695666604283 
minAngle=0.06716189480188284 
minEdgeLength=4.47213595499958 
sumAngle=45.15743785266318 
sumEdgeLength=3181.956077853515 
sumDistance=283083.02265907155 

0.0000000000 2.8269566660 0.0671618948 4.4721359550 45.1574378527 3181.9560778535 283083.0226590716
0: (103, 163),
1: (18, 173),
2: (71, 98),
3: (76, 90),
4: (100, 117),
5: (115, 200),
6: (77, 130),
7: (71, 84),
8: (0, 199),
9: (58, 73),
10: (0, 171),
11: (100, 73),
12: (38, 190),
13: (61, 94),
14: (86, 78),
15: (54, 100),
16: (68, 117),
17: (109, 195),
18: (101, 184),
19: (86, 150),
20: (35, 157),
21: (92, 161),
22: (63, 116),
23: (102, 179),
24: (93, 92),
25: (83, 161),
26: (71, 89),
27: (52, 143),
28: (131, 168),
29: (81, 91),
30: (87, 159),
31: (24, 185),
32: (107, 102),
33: (6, 196),
34: (161, 125),
35: (88, 73),
36: (66, 84),
37: (97, 172),
38: (48, 139),
39: (101, 97),
40: (70, 109),
41: (97, 89),
42: (65, 125),
43: (108, 96),
44: (22, 190),
45: (61, 39),
46: (96, 82),
47: (86, 68),
48: (103, 93),
49: (101, 103),
103 163 18 173 71 98 76 90 100 117 115 200 77 130 71 84 0 199 58 73 0 171 100 73 38 190 61 94 86 78 54 100 68 117 109 195 101 184 86 150 35 157 92 161 63 116 102 179 93 92 83 161 71 89 52 143 131 168 81 91 87 159 24 185 107 102 6 196 161 125 88 73 66 84 97 172 48 139 101 97 70 109 97 89 65 125 108 96 22 190 61 39 96 82 86 68 103 93 101 103 
0: (103, 163), 1: (18, 173), 2: (71, 98), 3: (76, 90), 4: (100, 117), 5: (115, 200), 6: (77, 130), 7: (71, 84), 8: (0, 199), 9: (58, 73), 10: (0, 171), 11: (100, 73), 12: (38, 190), 13: (61, 94), 14: (86, 78), 15: (54, 100), 16: (68, 117), 17: (109, 195), 18: (101, 184), 19: (86, 150), 20: (35, 157), 21: (92, 161), 22: (63, 116), 23: (102, 179), 24: (93, 92), 25: (83, 161), 26: (71, 89), 27: (52, 143), 28: (131, 168), 29: (81, 91), 30: (87, 159), 31: (24, 185), 32: (107, 102), 33: (6, 196), 34: (161, 125), 35: (88, 73), 36: (66, 84), 37: (97, 172), 38: (48, 139), 39: (101, 97), 40: (70, 109), 41: (97, 89), 42: (65, 125), 43: (108, 96), 44: (22, 190), 45: (61, 39), 46: (96, 82), 47: (86, 68), 48: (103, 93), 49: (101, 103), 
numberIntersection=0 minAngle=0.06716189480188284 sumAngle=45.15743785258468 minEdgeLength=4.47213595499958 sumEdgeLength=3181.956077853527 minDistance=2.82695666604283 sumDistance=283083.02265907236 
Time taken: 51.066 s
